dbt/include/trino/macros/crossdb/any_value.sql

```diff
@@ -1,3 +1,3 @@
-{% macro default__any_value(expression) -%}
+{% macro trino__any_value(expression) -%}
```

Comment:
Why are we doing this change?

Reply:
Any adapter-specific code should not reimplement the default but override the default by providing the`<adapter>__macro` implementation if necessary.